src/assets/scss/_mixins.scss

```diff
@@ -1,5 +1,12 @@
 @import './variables';
 
+@mixin icon-box {
+  padding: 10px;
+  border-radius: 6px;
+  color: $color-accent-primary;
+  background-color: rgba($color-accent-primary, 0.1);  
+}
```

Comment:
We use these styles in a few different places. I didn't want to copy them somewhere additional, so I moved them into a mixin. I could have made it an HTML class or a component instead, but this felt like the lightest-weight approach.